static/app/components/compactSelect/styles.tsx

```diff
@@ -117,6 +117,12 @@ export const SectionToggleButton = styled(Button)<{visible: boolean}>`
     opacity: 0;
     pointer-events: none;
   `}
+
+  li[role="rowgroup"]:hover &,
+  li[role="presentation"]:hover & {
+    opacity: 1;
+    pointer-events: all;
+  }
```

Comment:
Section toggle buttons ("Select All") will show up when hovering over the section, including the section header.
![image](https://user-images.githubusercontent.com/44172267/228978170-06653784-d3e6-45ee-ae97-136c5ef735bf.png)

Before, the buttons will only show up if one of the section's options have focus.

Reply:
Also I would be OKAY if we changed he UI for this and moved the checks to the left

Reply:
Decided to keep the checkboxes on the right side because of the page filters' weird hybrid selection mode. Here, the primary selection mode is singular. It's only when you click on the checkbox that the secondary selection mode — multiple — applies. 
Having the checkbox on the right, trailing side works better, I think, because that's where we put secondary actions. Putting it in the left, leading side might confuse the user into thinking that the primary selection mode is multiple.